src/js/actions/menu.js

```diff
@@ -428,7 +429,7 @@ define(function (require, exports) {
             actionModuleName = actionNameParts[0],
             actionName = actionNameParts[1],
             actionNameThrottled = actionName + "Throttled",
-            actionThrottled = this.flux.actions[actionModuleName][actionNameThrottled];
+            actionThrottled = objUtil.getPath(this.flux.actions, actionModuleName)[actionNameThrottled];
```

Comment:
Would this ideally be `FluxController.prototype.getModule` instead of `objUtil.getPath`?